### ISSUE TITLE
[Feature]: Ice effects on the cake 

### ISSUE THREAD

**Issue by Toy203** [gpstar81/GPStar-proton-pack#367]
### What would you like to add in terms of software changes?

This might be a stretch but figured to throw it in the ring. How about in the frozen state of FE (when Intensify is mashed too fast), instead of turning the cake ring light off, the light slows down and then it just could be going all led on with a light blue until it gets back on working mode to simulate the ice around it?
might look cool, might look wonky dunno. 

### Would this request involve any specific hardware?

Just the cake ring led. 

### Homework Completed

- [X] I did my part!

**Comment by DustinGrau** [comment]
Interesting. So basically once the cake "spins down" it just goes blue? The movie effect was to cause the pack to power down entirely so there was no movement (sparking) or system lights at all.

**Comment by Toy203** [comment]
Yes it was just a passing though, and it could not work at all to be honest for folks who has a transparent lid in the cake. But i thought at the look of the entire pack shut down and the cake just glowing light blue for few sec could be cool

**Comment by DustinGrau** [comment]
Implementation of a switch to have a "Ice Over" lighting option is easy, but the animation is harder. We can leave this request open, and consider it for a possible later release. We've been talking about refactoring the cyclotron code which is grown quite complex with the animations currently in use. It doesn't leave us much room to alter the sequence as it stands today.

**Comment by Toy203** [comment]
Well I like it for sure! XD

![Uploading
![20240911_130856](https://github.com/user-attachments/assets/9e183e06-d1ad-45a7-9413-d8e6f383eed3)
 20240911_130936.jpg…]()

**Comment by DustinGrau** [comment]
Yeah, I was thinking if we could do some kind of low flickering effect it would look good. Steady-on isn't likely to give the same effect, and there's lots of animations we could potentially do if we can just break free of the current set patterns.

**Comment by nomakewan** [comment]
Good news! I was working on getting fadeout working with the inner cyclotron and while working in there I got slime going. And once I got slime going I thought, "You know, this slime effect sure is similar to twinkling ice." So I've gone ahead and implemented this feature request in `6.1.Dev`, and it will be introduced as part of the next firmware release.

A demo of this can be seen here:

https://github.com/user-attachments/assets/5343f584-47c9-4db6-ab5c-3f5af1e71650
